### ISSUE TITLE
layoutComplete event fires early when no items are visible

### ISSUE THREAD

**Issue by EvanHerman** [metafizzy/isotope#885]
I have set up a test fiddle, and added an extra filter check box. This chec kbox has no items associated with it, so when it's set nothing is displayed.

You'll notice that when you click '.green' or any of the others the filtering animation is run and then the call back fires. If you click orange, you'll see that the callback event fires well before the animation is complete. If none are visible, and orange is checked off and you click 'shuffle' the correct number is returned.

I'm using this function to get the number of items visible, and running a function if none are. This small bug makes it hard to figure out if any are visible or not.

In short : 
When 0 items are visible after a filter, the callback event fires too soon

http://jsfiddle.net/QsCZx/244/

Thanks,
Evan

**Comment by desandro** [comment]
Thanks for reporting this issue! This sounds like a duplicate of #732. Sorry I do not have a solution for now.

**Comment by EvanHerman** [comment]
Thanks for responding. Sorry I didn't catch the duplicate ticket.

Best,
Evan
